katello_certs_tools/katello_ssl_tool.py

```diff
@@ -500,12 +535,12 @@ def genServerCert(password, d, verbosity=0):
 
     if verbosity > 2:
         if out:
-            print "STDOUT:", out
+            print("STDOUT:", out)
         if err:
-            print "STDERR:", err
+            print("STDERR:", err)
 
     # permissions:
-    os.chmod(server_cert, 0644)
+    os.chmod(server_cert, '0644')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)